src/applications/vaos/tests/e2e/vaos-cypress-helpers.js

```diff
@@ -654,9 +654,15 @@ export function initVaccineAppointmentMock({
       method: 'GET',
       url: '/vaos/v0/direct_booking_eligibility_criteria*',
       response: {
-        data: directEligibilityCriteria.data.filter(
-          facility => facility.id === 'covid',
-        ),
+        data: directEligibilityCriteria.data.map(facility => ({
```

Comment:
Previously, this was removing all facilities returned in the direct settings, which led to an application error because the mocked facilities data was returning a facility that had no settings.